tests/conftest.py

```diff
@@ -837,10 +839,11 @@ def task_extract_dw_employee_leave_complete(all_tasks_pipeline_name):
             [
                 {
                     "mit_id": "123456789",
-                    "appt_begin_date": Timestamp("2010-01-01 00:00:00"),
-                    "appt_end_date": datetime.datetime(2011, 12, 1, 0, 0),
+                    "appt_begin_date": Timestamp("2019-01-01 00:00:00"),
+                    "appt_end_date": datetime.datetime(2022, 12, 1, 0, 0),
+                    "position_id": "987654321",
                     "hr_appt_key": 123,
-                    "absence_date": Timestamp("2010-07-01 00:00:00"),
+                    "absence_date": Timestamp("2019-07-01 00:00:00"),
```

Comment:
This update, while not strictly required here, was just to better align mocked data in fixtures.